plugin/trino-http-query-listener/src/main/java/io/trino/plugin/httpquery/HttpQueryListenerEventType.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.httpquery;
+
+public enum HttpQueryListenerEventType
+{
+    CREATED,
```

Comment:
Lets be verbose. It is not that long still. So: QUERY_CREATED, QUERY_COMPLETED, SPLIT_COMPLETED